README.md

```diff
@@ -48,100 +48,96 @@ Your website/app must present information in ways people can recognise and use,
 
-* Do not require complex gestures or using more than one finger to do things. [More about 2.5.1]()
+* [Timing adjustable:]() If there is any time limit (like a session timeout), give people an easy way to extend it.
 
-* Controls do not make things happen as soon as they are touched. [More about 2.5.2]()
+* [Stop visual distractions:]() If any blinking, scrolling, animation or frequent updating happens automatically and last more than 5 seconds, give people a way to stop it.
 
-* When an interactive element has a visible label, make sure that the element's accessible name matches (or at least includes) what's on the label. [More about 2.5.3]()
+#### Do not cause seizures
 
-* If a feature uses shaking or tilting, make sure that it can also be activated in another way. And that responses to these motions can be turned off. [More about 2.5.4]()
+* [No three flashes:]() Do not show anything that flashes more than three times a second.
 
 ### 3. Easy to understand
 
 Your website/app must make information understandable, and make it easy for people to understand how to complete tasks.
 
-#### Make text readable and understandable
-
-* Identify the language that the content is written in. [More about 3.1.1]()
+#### Make text easy to understand
 
-* Identify any changes in language within the content. [More about 3.1.2]()
+* [Language of page:]() In code, identify the language that the content is written in. If the page uses several language, identify the language of each section.
 
-#### Make things appear and behave in consistent ways
+#### Make things appear and behave in consistent, predictable ways
 
-* Do not trigger any important or disorientating change when an element just receives focus. [More about 3.2.1]()
+* [Changes on focus:]() Make sure that just navigating to a element using the keyboard doesn't trigger any important or disorientating change (like opening a pop up).
 
-* Do not any trigger important or disorientating change when someone just changes the value of an form input. [More about 3.2.2]()
+* [Changes on input:]() Make sure that just changing the state of a form input (like a dropdown menu) does not trigger any important or disorientating change.
 
-* Make sure that navigation elements that appear on multiple pages look and behave consistently. [More about 3.2.3]()
+* [Consistent navigation:]() Make sure that navigation elements that appear on multiple pages look and behave consistently.
 
-* Make sure that features that appear on multiple page are identified consistently. [More about 3.2.4]()
+* [Consistent feature names:]() Make sure that features that appear on multiple page are identified consistently.
 
 #### Help people avoid and correct mistakes
 
-* When someone makes a mistake, provide an error message in text and make it obvious where the mistake was made. [More about 3.3.1]()
+* [Error identification:](): When someone makes a mistake, provide an error message in text and make it obvious where the mistake was made.
```

Comment:
This may be answered in the detail page, but wondering how this is expected to behave for screen readers where it might not be clear that error message in text has been displayed somewhere. Should the focus be moved, or just an announcement made?

Reply:
Different people have different opinions on the best experience for screen reader users.

For Web, the traditional model is that you'd like users hit the submit button on a form even if the form contains errors. Then, the page would reload, and hence the keyboard focus would go back to the top of the page. The `title` of the page would start with 'Error -', and the user would go down the form elements and find the errors. There'd be a box summarising all the errors at the top of the form, with links to the errors. This is how the [GDS Error Summary](https://design-system.service.gov.uk/components/error-summary/) behaves, and this experience works very well.

But when an form is validated with javascript on the client side, there's debate about how to best handle this. Léonie Watson says that we should definitely not move the keyboard focus to any error. Instead, we should keep the keyboard focus on the submit button. But use something like `aria-describedby` on the submit button, or a live region, to signal that there are errors. Then, the screen reader user would jump back up the controls on the form. Each of the control would have an error described in line, and that error description would be programatically linked with the input field (again with `aria-describedby`. This is a solid approach too.

I'd love to know how to best handle things on iOS.

Reply:
In terms of how to summarise this success criterion, I don't think that we need to mention that the errors need to be programatically associated with the inputs, because:
* the official wording of [3.3.1 'Error identification'](https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-identified.html) doesn't either
* that point is covered by [1.3.1 'Info and relationships'](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)